### DISCUSSION TITLE
i have a problem in my contributions graph

### DISCUSSION THREAD

**Discussion by Y-askour** [community/community#20544] [Other Feature Feedback, Questions, & Ideas]
I have a problem with my contributions graph all the contributions I made this year they didn't count them 
is there any solution to this ??

**Comment by jfullstackdev** [comment]
just a quick observation, your commits are not verified

![image](https://user-images.githubusercontent.com/47092464/179045989-26d900f0-1009-454e-8214-ce8ad85090f1.png)

**Comment by LuigiImVector** [comment]
maybe you commit to the private repository without activating the "private contributing" counter:

[image](https://user-images.githubusercontent.com/59611597/179047758-c5c283f7-c809-433c-ac78-8b99f8183e65.png)

https://skyline.github.com/Y-askour/2022: seems that there are all the commits
